Assignment2/sebufer-SergioBujanFernandez.json

```diff
@@ -0,0 +1,23 @@
+"@context":{
+	"rdfs":"http:www.oeg-upm.net/ontologies/class#",
+	"includes": "rdf:includes",
+	"hasMeasurament":  "rdf:hasMeasurament",
+	"hasTemperatura":  "rdf:integer",
+	"atTime":  "rdf:dataTime",
+	"hasOwner": "rdf:hasOwner",
+	"hasName":  "rdf:string"

```

Comment:
The prefix "rdf" is not defined. Also, try to use another name as rdf is reserved.
atTime, hasTemperatura, and hasName are datatypes thus you have to specify its attribute name and its type. In addition, dataTime, string and integer might be defined with the xsd vocabulary instead of rdf, thus you should define it as well and then use it. 
For defining datatypes you should have something like 
"atTime":{"@id":"vocab:atTime","@type":"xsd:dataTime"}